app/controllers/line_items_controller.rb

```diff
@@ -42,7 +42,8 @@ def destroy_with_lock(item)
       item.destroy
       order.update_shipping_fees!
       order.update_payment_fees!
-      order.update_distribution_charge!
+      order.update_order_fees!
+      order.update!
```

Comment:
In this case we're deleting a line item from an order that is probably complete. We don't need to delete and re-create any fees that might already exist on other line items, we only need to update order-level fees (if any).